staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/interfaces.go

```diff
@@ -35,7 +35,7 @@ type Source interface {
 // variants of the object and old object.
 type VersionedAttributes struct {
 	// Attributes holds the original admission attributes
-	Attributes admission.Attributes
+	admission.Attributes
```

Comment:
is this change intended?

Reply:
it is intended... it allows passing versionedattributes to ShouldCallHook as admission.Attributes and overriding the GetObject() method